common/enablers/enableEpochsHandler.go

```diff
@@ -39,6 +41,330 @@ func (handler *enableEpochsHandler) EpochConfirmed(epoch uint32, _ uint64) {
+	case core.MultiESDTTransferFixOnCallBackFlag:
+		return epoch >= handler.enableEpochsConfig.MultiESDTTransferFixOnCallBackOnEnableEpoch
+	case core.OptimizeGasUsedInCrossMiniBlocksFlag:
+		return epoch >= handler.enableEpochsConfig.OptimizeGasUsedInCrossMiniBlocksEnableEpoch
+	case core.CorrectFirstQueuedFlag:
+		return epoch >= handler.enableEpochsConfig.CorrectFirstQueuedEpoch
+	case core.DeleteDelegatorAfterClaimRewardsFlag:
+		return epoch >= handler.enableEpochsConfig.DeleteDelegatorAfterClaimRewardsEnableEpoch
+	case core.RemoveNonUpdatedStorageFlag:
+		return epoch >= handler.enableEpochsConfig.RemoveNonUpdatedStorageEnableEpoch
+	case core.OptimizeNFTStoreFlag,
+		core.SaveToSystemAccountFlag,
+		core.CheckFrozenCollectionFlag,
+		core.ValueLengthCheckFlag,
+		core.CheckTransferFlag:
+		return epoch >= handler.enableEpochsConfig.OptimizeNFTStoreEnableEpoch
+	case core.CreateNFTThroughExecByCallerFlag:
+		return epoch >= handler.enableEpochsConfig.CreateNFTThroughExecByCallerEnableEpoch
+	case core.StopDecreasingValidatorRatingWhenStuckFlag:
+		return epoch >= handler.enableEpochsConfig.StopDecreasingValidatorRatingWhenStuckEnableEpoch
+	case core.FrontRunningProtectionFlag:
+		return epoch >= handler.enableEpochsConfig.FrontRunningProtectionEnableEpoch
+	case core.PayableBySCFlag:
+		return epoch >= handler.enableEpochsConfig.IsPayableBySCEnableEpoch
+	case core.CleanUpInformativeSCRsFlag:
+		return epoch >= handler.enableEpochsConfig.CleanUpInformativeSCRsEnableEpoch
+	case core.StorageAPICostOptimizationFlag:
+		return epoch >= handler.enableEpochsConfig.StorageAPICostOptimizationEnableEpoch
+	case core.ESDTRegisterAndSetAllRolesFlag:
+		return epoch >= handler.enableEpochsConfig.ESDTRegisterAndSetAllRolesEnableEpoch
+	case core.ScheduledMiniBlocksFlag:
+		return epoch >= handler.enableEpochsConfig.ScheduledMiniBlocksEnableEpoch
+	case core.CorrectJailedNotUnStakedEmptyQueueFlag:
+		return epoch >= handler.enableEpochsConfig.CorrectJailedNotUnstakedEmptyQueueEpoch
+	case core.DoNotReturnOldBlockInBlockchainHookFlag:
+		return epoch >= handler.enableEpochsConfig.DoNotReturnOldBlockInBlockchainHookEnableEpoch
+	case core.AddFailedRelayedTxToInvalidMBsFlag:
+		return epoch < handler.enableEpochsConfig.AddFailedRelayedTxToInvalidMBsDisableEpoch
+	case core.SCRSizeInvariantOnBuiltInResultFlag:
+		return epoch >= handler.enableEpochsConfig.SCRSizeInvariantOnBuiltInResultEnableEpoch
+	case core.CheckCorrectTokenIDForTransferRoleFlag:
+		return epoch >= handler.enableEpochsConfig.CheckCorrectTokenIDForTransferRoleEnableEpoch
+	case core.FailExecutionOnEveryAPIErrorFlag:
+		return epoch >= handler.enableEpochsConfig.FailExecutionOnEveryAPIErrorEnableEpoch
+	case core.MiniBlockPartialExecutionFlag:
+		return epoch >= handler.enableEpochsConfig.MiniBlockPartialExecutionEnableEpoch
+	case core.ManagedCryptoAPIsFlag:
+		return epoch >= handler.enableEpochsConfig.ManagedCryptoAPIsEnableEpoch
+	case core.ESDTMetadataContinuousCleanupFlag,
```

Comment:
on the original code, I see this epoch flag tight only to `esdtMetadataContinuousCleanupFlag` and `changeDelegationOwnerFlag`. Is this correct?

Reply:
same as above, duplicated for consistency: https://github.com/multiversx/mx-chain-go/blob/rc/v1.6.0/common/enablers/epochFlags.go#L610